pages/admin/index.tsx

```diff
@@ -41,30 +47,34 @@ export default function Admin() {
         reviewItem!.displayed = !reviewItem!.displayed
         setRevsArray(revsArrayCopy)
```

Comment:
You could consider calling this in the `.then` of the PUT request so the user only gets the visual feedback about their update (i.e. changing the colour of the card) once it has been successful.

Reply:
Oh I wanted to speed it up, so I update the UI first before updating the database. Because it was really laggy visually, since it had to wait for the PUT then GET again. But I guess if I do it in the `.then`, it would only wait for the PUT request not the GET? Still there will probably be a delay which annoys me

Reply:
If you do it in the `.then` it will only wait for the PUT but yes there will still be a delay, I guess it's personal preference. Personally I would make a loading icon appear next to it after you press it but before the PUT's gone through and then change the colour and remove the loading icon once it's been successful. It's up to you.